modules/admin-ui-frontend/app/scripts/shared/partials/modals/event-details.html

```diff
@@ -1,51 +1,120 @@
-<section ng-show="open" ng-keyup="keyUp($event)" tabindex="1" class="modal modal-animation ng-hide modal-open" id="event-details-modal" ng-controller="EventCtrl">
-
+<section
```

Comment:
You essentially reformatted the whole file. This makes it more or less impossible to track what actually changed in a given commit. Everyone has his/her own style preferences, but we can't change everything every time we touch a file.

Can you update the patch and change only the part you want to change?

Reply:
I'm really sorry. I forgot disable the prettier extension.